utils/configureEnv.js

```diff
@@ -0,0 +1,9 @@
+const dotenv = require('dotenv');
+
+module.exports = () => {
+    if (process.env.ENV === 'production') {
+        dotenv.config();
+    } else {
+        dotenv.config({ path: `.env.${process.env.ENV}` });
```

Comment:
if wthe production env file is .env.production then just the else should cover all cases